src/applications/vaos/utils/constants.js

```diff
@@ -477,8 +477,10 @@ export const DATE_FORMATS = {
   // e.g. 2025-05-06T21:00:00-05:00"
   ISODateTimeLocal: "yyyy-MM-dd'T'HH:mm:ssXXX",
   // iCalendar RFC 5545
+  // e.g. 20250506T225403
+  iCalDateTime: "yyyyMMdd'T'HHmmss",
   // e.g. 20250506T225403Z
-  iCalDateTimeUTC: "yyyyMMdd'T'HHmmss'Z'",
+  iCalDateTimeUTC: "yyyyMMdd'T'HHmmssXXX",
```

Comment:
This is explicitly used by `generateICS` to ensure the time is formatted in UTC, we shouldn't change this. If needed we should use a different value like `iCalDateTimeOffset` to indicate that the string may contain +/- offsets